packages/paypal-commerce-integration/src/mocks/paypalCommerceRatePayMocks.ts

```diff
@@ -0,0 +1,109 @@
+import {
+    Cart,
+    Checkout,
+    Consignment,
+    Coupon,
+    Customer,
+    Discount,
+    GiftCertificate,
+    Tax,
+} from '@bigcommerce/checkout-sdk';
+
+export function getPaypalCommerceRatePayMethodMock() {
+    return {
+        clientToken: 'token',
+        config: {
+            cardCode: true,
+            displayName: 'RatePay',
+            enablePaypal: undefined,
+            hasDefaultStoredInstrument: false,
+            helpText: '',
+            is3dsEnabled: undefined,
+            isHostedFormEnabled: false,
+            isVaultingCvvEnabled: undefined,
+            isVaultingEnabled: false,
+            isVisaCheckoutEnabled: undefined,
+            logo: undefined,
+            merchantId: '',
+            requireCustomerCode: false,
+            testMode: true,
+        },
+        gateway: 'paypalcommercealternativemethods',
+        id: 'ratepay',
+        initializationData: {
+            attributionId: 'BC_id',
+            availableAlternativePaymentMethods: [],
+            clientId: 'asd123',
+            enabledLocalPaymentMethods: [],
+            intent: 'capture',
+            isComplete: false,
+            isDeveloperModeApplicable: false,
+            isHostedCheckoutEnabled: true,
+            isInlineCheckoutEnabled: false,
+            isPayPalCreditAvailable: true,
+            isVenmoEnabled: true,
+            merchantId: 'XdasdQWe1123',
+            orderId: undefined,
+            shouldRenderFields: true,
+            showOnlyOnMobileDevices: false,
+        },
+        initializationStrategy: {
+            type: 'not_applicable',
+        },
+        logoUrl: 'http://logo_url_path',
+        method: 'braintree',
+        nonce: undefined,
+        returnUrl: undefined,
+        supportedCards: [
+            'PAYPAL',
+            'PAYPALCREDIT',
+            'VISA',
+            'MC',
+            'AMEX',
+            'DISCOVER',
+            'VENMO',
+            'APPLEPAY',
+        ],
+        type: 'PAYMENT_TYPE_API',
+    };
+}
+
+export function getCheckoutMock(): Checkout {
```

Comment:
As far as I know you can use getCheckoutMock from `payment-integration-unit-tests` package. Can you double check it?

Reply:
fixed